src/main.ts

```diff
@@ -43,7 +54,9 @@ async function run(): Promise<void> {
       body ||
       Buffer.from(
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        (pullRequestTemplate as any).content,
+        pullRequestTemplate
```

Comment:
It might make more sense to move this condition up a line:

```
body || pullRequestTemplate
  ? Buffer.from(pullRequestTemplate.content, 'base64')
  : ''
```

Since `Buffer.from('')` always return `''`.

Reply:
Not 100% sure why, but tests are failing when I change it, so I'll just leave as is